doc/source/serve/tutorials/batch.rst

```diff
@@ -0,0 +1,119 @@
+.. _serve-batch-tutorial:
```

Comment:
Will you please link this from the batching bit in the [advanced](https://docs.ray.io/en/master/serve/advanced.html#batching-to-improve-performance) section?